include/rtosc/bundle-foreach.h

```diff
@@ -64,7 +64,7 @@ void bundle_foreach(const struct Port& p, const char* name, char* old_end,
     for(unsigned i=0; i<max; ++i)
     {
         const char* name2_2 = name;
-        pos2 = pos + sprintf(pos,"%d",i);
+        pos2 = pos + snprintf(pos,16,"%d",i);  // TODO: check length is >= 16
```

Comment:
This feels cheaty since this commit doesn't update the API of bundle_foreach to explicitly accept a true length.